gamestonk_terminal/portfolio/portfolio_helper.py

```diff
@@ -60,3 +60,23 @@ def clean_name(name: str) -> str:
         A cleaned value
     """
     return name.replace("beta_", "").upper()
+
+
+def get_fraction(n: float, d: float) -> str:
+    """Turn two numbers into a fraction
+
+    Parameters
+    ----------
+    n : float
+        The numerator
+    d : float
+        The denominator
+
+    Returns
+    ----------
+    text : str
+        A fraction as a string
+    """
+    if d > 0:
```

Comment:
I think here should be if `d != 0` as you can still get the fraction if d is a negative number

Reply:
This if is to prevent negative numbers from returning. The only value that can be negative is the Treynor ratio, and the Treynor ratio becomes nonsensical if the denominator is negative.

Reply:
Ok, so this is something that you should bear in mind. The name of the function is `get_fraction` and therefore if I was looking for a similar function I would think "Ohh, ok, look this function gives me the fraction". But in this case you are doing a specific manipulation (not accepting negative values to be returned).

So, I would proceed in 1 of 2 ways:
- Either rename the function so people know that this is for your specific case, and mention on the docstring that it doesn't accept negative denominators.
- Allow the function to just return the division value with 2 decimal places, and then when you use it in the context you are using have the check for negative denominator before calling the function.